src/main/kotlin/com/openlattice/shuttle/Shuttle.kt

```diff
@@ -463,4 +545,25 @@ class Shuttle(
 
         return if (keyValuesPresent) ApiUtil.generateDefaultEntityId(key.stream(), properties) else ""
     }
+
+    private fun storeLog(flightName: String, log: String, timestamp: OffsetDateTime, status: IntegrationStatus, jobId: UUID) {
+        val logEntitySetId = logEntitySet.id
+        val entityId = logEntitySetId.toString() + flightName + jobId + timestamp.toString()
```

Comment:
I really wish there was something between your code above and the following that indicates string-ness of entityId
`"$logEntitySetId$flightName$jobId$timestamp"`
No changes needed